ui/src/message_popup/experience_page.jsx

```diff
@@ -117,24 +98,32 @@ export default React.createClass({
     this.setState(this.getInitialState());
   },
 
+  onSaveScaffoldingAndSession(scaffolding, gameSession){
+    this.setState({
+      scaffolding: {
+        ...scaffolding
+      },
+      gameSession: {
+        ...gameSession,
+        sessionId: uuid.v4(),
+        questionsAnswered: 0,
+        responseTimes: []
+      },
+      hasStarted: true
```

Comment:
Would it make sense to have the session be created here, and then we don't need a separate state value for `hasStarted` since we can just check "is there a gameSession"?